ChangeLog

```diff
@@ -0,0 +1,237 @@
+version <unreleased>
+
+version 2016.07.28
+
+Fixed/improved extractors
+- shared (#10170)
+- soundcloud (#10179)
+- twitch (#9767)
+
+version 2016.07.26.2
+
+Fixed/improved extractors
+- smotri
+- camdemy
+- mtv
+- comedycentral
+- cmt
+- cbc
+- mgtv
+- orf
+
```

Comment:
With two blank lines between versions it's easier to visually distinguish where one ends and another starts.